blog/2024-07-31-open-planning-r24.12.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Tractus-X Open Planning R24.12
+description: The Eclipse Tractus-X Open Planning Days - Empowering the Catena-X Data Space
+slug: open-planning-r24-12
+date: 2024-04-16T11:00
+hide_table_of_contents: false
+authors:
+  - name: Stephan Bauer
+    title: Platform Domain Manager
+    url: https://github.com/stephanbcbauer
+    image_url: https://github.com/stephanbcbauer.png
+    email: stephan.bauer@catena-x.net
+---
+
+![eclipse tractus x logo](@site/static/img/Tractus-X_openplanning.png)
```

Comment:
For static content, there is no need to reference via `@site`.
See [docusaurus docs](https://docusaurus.io/docs/static-assets#in-markdown)

`/img/Tractus-X_openplanning.png` is enough

Reply:
Changed it, thanks for the hint. It still works. :)